po/es.po

```diff
@@ -449,12 +455,12 @@ msgid ""
 "  Library prefix:    %s\n"
 "  Compiled for:      %s\n"
 msgstr ""
-"Opciones de compilado:\n"
-"  \"Release\" versión:      %s (%s)\n"
-"  BinReloc habilitado:    %s\n"
-"  Prefijo de datos:       %s\n"
-"  Prefijo de bibliotecas: %s\n"
-"  Compilado en:           %s\n"
+"Opciones de compilación:\n"
+"  Versión de producción: %s (%s)\n"
```

Comment:
"release" means "mark version" from versions.. literall translation are "liberacion" but the meaning are "version de lanzamiento"

Reply:
It also means "Versión publicada" or "Versión final",  i. e. "Versión de producción" because is the version aimed to be reliable for production. But "lanzamiento" is also common, it's ok.

Reply:
**OK** let them as you translated!